minigotchi-ESP32/display.cpp

```diff
@@ -62,59 +66,67 @@ void Display::startScreen() {
       delay(100);
       tft.setTextSize(2); // Set text size
       delay(100);
-    } else if (Config::screen == "T_DISPLAY_S3") {
-      tft.begin();
+    } else if (Config::screen == "M5StickCP" || Config::screen == "M5StickCP2" || Config::screen ==  "M5Cardputer") {  // New condition for M5StickC Plus
+      tft.begin(); // Initialize TFT_eSPI library
       delay(100);
-      tft.setRotation(1);
+      tft.setRotation(1); // Set display rotation if needed
       delay(100);
-      tft.fillScreen(TFT_BLACK);
+      tft.fillScreen(TFT_BLACK); // Fill screen with black color
       delay(100);
-      tft.setTextColor(TFT_WHITE);
+      tft.setTextColor(TFT_WHITE); // Set text color to white
       delay(100);
-      tft.setTextSize(2);
+      tft.setTextSize(2); // Set text size)
       delay(100);
     } else {
       // use wemos shield by default
       adafruit_display = new Adafruit_SSD1306(WEMOS_OLED_SHIELD_OLED_RESET);
-      if (!adafruit_display->begin(SSD1306_SWITCHCAPVCC,
-                                   0x3C)) { // initialize with the I2C addr 0x3C
+      if (!adafruit_display->begin(SSD1306_SWITCHCAPVCC, 0x3C)) { // initialize with the I2C addr 0x3C
         Serial.println(F("SSD1306 allocation failed"));
         return;
       }
-      delay(100); // initialize w/ delays to prevent crash
+      delay(5); // initialize w/ delays to prevent crash
       adafruit_display->display();
-      delay(100);
+      delay(5);
       adafruit_display->clearDisplay();
-      delay(100);
+      delay(5);
       adafruit_display->setTextColor(WHITE);
-      delay(100);
+      delay(5);
     }
   }
 }
 
 void Display::cleanDisplayFace(String text) {
   if (Config::display) {
-    if (Config::screen ==
-        "CYD") { // Check if the screen configuration is set to "CYD" and
-      // execute the corresponding code
-      tft.fillScreen(TFT_BLACK); // Clear screen with black color
+    if (Config::screen == "CYD") {   // Check if the screen configuration is set to "CYD" and execute the corresponding code
+      tft.fillScreen(TFT_BLACK);  // Clear screen with black color
       delay(5);
-      tft.setTextColor(TFT_WHITE); // Set text color to WHITE
+      tft.setTextColor(TFT_WHITE);  // Set text color to WHITE
       delay(5);
-      tft.setCursor(0, 5); // Set cursor position
+      tft.setCursor(0, 5);  // Set cursor position
       delay(5);
-      tft.setTextSize(4); // Set text size
+      tft.setTextSize(4);  // Set text size
       delay(5);
-      tft.println(text); // Print text
+      tft.println(text);  // Print text
       delay(5);
-    } else if (Config::screen == "T_DISPLAY_S3") {
-      tft.fillScreen(TFT_BLACK);
+    } else if (Config::screen == "TTGO_T_DISPLAY") { // New condition for TTGO_T_DISPLAY
```

Comment:
im not sure if we have different displays, but I would prefer that we keep the settings I put down for it, I tried to adhere to the name of the product kind of. As for screen coordinates, I don't really care as long as it's *mostly* pwnagotchi-like and it looks similar to the other displays

Reply:
To make it simpler and more streamlined I would suggest it just be T-Display or something similar. As your sketch worked on my 1.14in t-display. Ppl will see the s3 and think it only runs on that.